src/current/_includes/releases/v23.2/v23.2.0-rc.1.md

```diff
@@ -0,0 +1,60 @@
+- Added support for [`IMPORT INTO`](../v23.2/import-into.html) a table that has columns typed as arrays of user-defined types (like [`ENUM`](../v23.2/enum.html)). Tables that use multiple user-defined types with the same name but different schemas are still unsupported. [#116360][#116360]
+- The [`SELECT FOR UPDATE`](../v23.2/select-for-update.html) implementation used under Read Committed isolation (and under [Serializable isolation](../v23.2/demo-serializable.html) when [`optimizer_use_lock_op_for_serializable`](../v23.2/set-vars.html) is set to `true`) now locks all [column families](../v23.2/column-families.html) instead of only the first column family. [#116828][#116828]
+
+<h3 id="v23-2-0-rc-1-operational-changes">Operational changes</h3>
+
+- The [cluster settings](../v23.2/cluster-settings.html) `storage.single_delete.crash_on_invariant_violation.enabled` and `storage.single_delete.crash_on_ineffectual.enabled` now default to `true` because these can be indicative of a serious bug that could corrupt data. [#116604][#116604]
+
+<h3 id="v23-2-0-rc-1-command-line-changes">Command-line changes</h3>
+
+- Added the command [`cockroach debug enterprise-check-fips`](../v23.2/cockroach-commands.html) that diagnoses errors in [FIPS](../v23.2/fips.html) deployments. [#116281][#116281]
+- Added the flag `--enterprise-require-fips-ready` that can be run with any [CockroachDB command](../v23.2/cockroach-commands.html) to prevent startup if certain prerequisites for [FIPS](../v23.2/fips.html) compliance are not met. [#116281][#116281]
+
+<h3 id="v23-2-0-rc-1-db-console-changes">DB Console changes</h3>
+
+- Updated the **CPU Time** label to **SQL CPU Time** in the [DB Console](../v23.2/ui-runtime-dashboard.html) and added clarification to its tooltip. [#116450][#116450]
+- Removed the ID from the event description in the [DB Console](../v23.2/ui-runtime-dashboard.html) when it is `undefined`. [#116519][#116519]
+
+<h3 id="v23-2-0-rc-1-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused node crashes and panics when running [`INSERT`](../v23.2/insert.html) queries on [`REGIONAL BY ROW`](../v23.2/table-localities.html#regional-by-row-tables) tables with `UNIQUE` constraints or indexes. The bug is only present in version v23.2.0-beta.1. [#116343][#116343]
+- [`UPDATE`](../v23.2/update.html), [`UPSERT`](../v23.2/upsert.html), and [`INSERT ON CONFLICT`](../v23.2/insert.html#on-conflict-clause) queries are now disallowed under Read Committed isolation when the table contains a [`CHECK` constraint](../v23.2/check.html) involving a [column family](../v23.2/column-families.html) that is updated, and the `CHECK` constraint also involves a column family that is **not** updated, but **is** read. This is a temporary fix to prevent possible violation of the `CHECK` constraint, and the restriction will be lifted in the future. [#116429][#116429]
```

Comment:
```suggestion
- [`UPDATE`](../v23.2/update.html), [`UPSERT`](../v23.2/upsert.html), and [`INSERT ON CONFLICT`](../v23.2/insert.html#on-conflict-clause) queries are now disallowed under Read Committed isolation when the table contains a [`CHECK` constraint](../v23.2/check.html) involving a [column family](../v23.2/column-families.html) that is updated, and that `CHECK` constraint also involves a column family that is **not** updated, but **is** read. This restriction is a temporary fix to prevent possible violation of the `CHECK` constraint. However, it is important to note that this restriction will be lifted in the future. [#116429][#116429]
```